src/video_core/renderer_opengl/renderer_opengl.cpp

```diff
@@ -150,6 +150,8 @@ void RendererOpenGL::SwapBuffers() {
     {
         auto aggregator = Common::Profiling::GetTimingResultsAggregator();
         aggregator->AddFrame(profiler.GetPreviousFrameResults());
+        if (VideoCore::g_toggle_framelimit_enabled)
+        FrameLimiter(aggregator->GetAggregatedResults().frame_time.avg, 60);
```

Comment:
but does citra adapt the renderer to the game if it runs at 30fps,ie, without vsync and using a good framelimiter, will overlays show 30fps as the fps if the game runs at 100%speed.